esi_leap/resource_objects/ironic_node.py

```diff
@@ -45,6 +45,10 @@ def __init__(self, ident):
         else:
             self._node = None
             self._uuid = ident
+        self._node_name = None
+        self._provision_state = None
+        self._power_state = None
+        self._node_properties = None
```

Comment:
Ah - notifications won't work on dummy/fake nodes unless they have these values as well, correct? I'm not sure we want to set these values however; adding a comment above.

Reply:
Yes. Notification only works on ironic nodes currently.